README.md

```diff
@@ -87,10 +87,14 @@ php artisan vendor:publish --provider="Spatie\Permission\PermissionServiceProvid
 If you're using UUIDs or GUIDs for your `User` models you can update the `create_permission_tables.php` migration and replace `$table->morphs('model')` with:
 
 ```php
-$table->uuid('model_id');
-$table->string('model_type');
+$table->uuid(config('permission.column_names.model_morph_key'));
+$table->string(config('permission.column_names.model_morph_type'));
```

Comment:
Did you mean to also include `model_morph_type` in the `config` file and in the migration? 
The README example here will get nulls if a fallback isn't provided and if it's not defined in the config file.

Reply:
Great timing :)  I see you addressed this in da20346
https://github.com/spatie/laravel-permission/pull/827/commits/da2034676fe954c7407440e0cebb83090a034ebd

Reply:
No, sorry, that was an attempt to make the other column configurable too, but Laravel will not allow a custom type column name on MorphToMany relationships.

Hence I reverted the change in the code. But forgot to do it in the docs.

Reply:
Er, well, partially. The unused config key is still mentioned here in the readme.

Evidently you're actively looking at it ... so I'll review it a bit later ;)